dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormWithToggle.vue

```diff
@@ -65,9 +67,10 @@
         <FormKit
           :disabled="!dataPointIsAvailable"
           type="number"
+          data-test="valueAsPercentage"
```

Comment:
valueAsPercentage is only accurate if the value is a percentage

Reply:
I just renamed it to data-test="value"